graylog2-web-interface/src/components/graylog/Modal.jsx

```diff
@@ -16,6 +16,27 @@ const Modal = styled(BootstrapModal)(({ theme }) => css`
       border-top-color: ${theme.color.secondary.tre};
     }
   }
+
+  .modal-title {
+    font-size: 21px;
+  }
+
+  .modal-dialog {
+    margin-top: 55px;
+  }
+
+  .modal-backdrop {
+    height: 100000%;  /* yes, really. this fixes the backdrop being cut off when the page is scrolled. */
+    z-index: 1030;
+  }
+
+  form {
+    margin-bottom: 0;
+  }
+
+  .modal-body .form-group {
```

Comment:
Can we delete this from the global styles? https://github.com/Graylog2/graylog2-server/blob/f0bb3dc08c0b1ec64445c6eea15f04351cdd50fd/graylog2-web-interface/src/theme/GlobalThemeStyles.jsx#L1459